routes/web.php

```diff
@@ -28,5 +29,7 @@
 Route::post('/logout', [LoginController::class, 'logout'])->name('logout')->middleware('auth');
 
 Route::middleware('auth')->group(function () {
+    Route::get('/register/new-team', [NewTeamController::class, 'index'])->name('new_team');
+    Route::post('/create-new-team', [NewTeamController::class, 'create'])->name('new_team.create');
```

Comment:
It would be nice to rename the pages and endpoints to follow the RESTful conventions.

This would be `POST teams`